evaluator/src/main/kotlin/RuleSet.kt

```diff
@@ -51,13 +51,13 @@ class RuleSet(val ortResult: OrtResult) {
      */
     fun packageRule(name: String, configure: PackageRule.() -> Unit) {
         val packages = ortResult.analyzer?.result?.let { analyzerResult ->
-            analyzerResult.projects.map { it.toPackage() } + analyzerResult.packages.map { it.pkg }
+            analyzerResult.projects.map { it.toPackage().toCuratedPackage() } + analyzerResult.packages
         }.orEmpty()
 
         packages.forEach { pkg ->
-            val detectedLicenses = licenseFindings[pkg.id].orEmpty().keys.toList()
+            val detectedLicenses = licenseFindings[pkg.pkg.id].orEmpty().keys.toList()
```

Comment:
Maybe rename the `pkg` lambda variable to `curPkg` now to make this look slightly nicer?

Reply:
Done.